src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -403,6 +405,15 @@ async function validate_bucket_args(config_fs, data, action) {
             }
         }
     }
+
+    //if there's a change to the bucket's notifications, we need to test them
+    //if one of the specified notifications fail, we need to fail the user's request
+    if (user_input.notifications) {
+        const test_notif_err = await notifications_util.test_notifications(user_input, config_fs.connections_dir_path);
+        if (test_notif_err) {
+            throw_cli_error(ManageCLIError.InvalidArgument, "Failed to update notifications", test_notif_err);
```

Comment:
Maybe we can use a more specific error than `InvalidArgument`?

Reply:
S3 errors don't have a specific error for this, I'd like to follow suit.

Reply:
Sure, just notice that It's a `ManageCLIError`, so you have more flexibility in giving the precise error you need.